File: java/demo/src/main/java/SeleniumProject/Activity1.java
package SeleniumProject;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
public class Activity1 {

    public static void main(String[] args){
        WebDriver driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/lms");
        System.out.println("Page Title: "+driver.getTitle());

        driver.quit();



    }
    
}

File: selenium/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.select import Select
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net")
    print(driver.title)
    click_on = driver.find_element(By.LINK_TEXT,"About Us").click()
    print(driver.title)
    driver.quit()
